microsoft/PowerToys
# Fix for IDS_IMAGERESIZER bug: Added unlocalized english strings to all string tables in ImageResizer

## Summary of the Pull Request
The current fix is to just add all the unlocalized English strings (in this case the strings that we added), to all the string tables of different languages as well. Based on my investigation there is no way to have some global strings which are in English while also have a string table with some localized strings (in case of IR the "Resize pictures" is already localized in the rc file). The only solutions until we have translated versions are either add the english version to all the string tables or comment out the translated string tables so only english is supported

## Changes
Since the diff view for the rc file is not visible, the changes made in the file are like this.
![image](https://user-images.githubusercontent.com/32061677/78171555-771ae380-7409-11ea-896d-0cbf1b761db5.png)
For each language's string table (Chinese in the above image for example) I added the last 4 english strings (since we don't have translated versions of them).


## PR Checklist
* [X] Applies to #1795 


## Validation Steps Performed
Build and installed with MSI. Changed OS language to French to verify that those strings appear correctly.
